services/web/src/components/form-fields/Password.js

```diff
@@ -0,0 +1,14 @@
+import { TextInput } from '@mantine/core';
+
+export default function PasswordField(props) {
+  return (
+    <TextInput
+      name="password"
+      type="password"
+      label="Password"
+      placeholder="Password"
+      autoComplete="new-password"
+      {...props}
+    />
```

Comment:
Mantine has a password field

Reply:
I'm OK to change to that but in general I want MORE specific form fields than we have now because we keep re-inventing the wheel every project